### ISSUE TITLE
TineMCE 3.5.10 (latest) remove(); [jQuery] on IE causes focus lost. LOST!!! Need to $('input').focus(); to activate. Otherwise cannot click any field

### ISSUE THREAD

**Issue by TinyMCE-User** [tinymce/tinymce#1283]
TineMCE 3.5.10 (latest) remove(); [jQuery] on IE causes focus lost. LOST!!! Need to $('input').focus(); to activate. Otherwise cannot click any field when div is removed by jquery or tinymce destroyed. 

PROBLEM WITH FOCUS.

PLEASE FIX!

read below my comments:

(Please make tinymce work with cross domains)

// nasty bug in IE with TinyMCE 3.5.10, sorry! Without that is unable to click to input field when you close the window that has tinymce WYSIWYG initialized.
// Focus is still in tinymce somewhere, and JS crash when that iframe is removed but focus leaves.
// tried to create invisible field, but didnt work :(((
// IE focus() problem [discovered]
// tinyMCE is not a part of Ajaxel CMS, we cant change that thing.
// Also there is a tinyMCE problem with cross domains. Why all Ajaxel scripts are working but not tinymce, are they sooo STUPID?! They want to be? LOL!
// Many google results for that. Check that if don't believe.
if ($.browser.msie) {
    var input = $('input,textarea').focus();
}

this.wins[win_id].win.remove();
this.wins[win_id].dock.remove();
delete this.wins[win_id];
this.scroll(true);
this.unblock();

// Keep removing focus for crushy result!
// Bad solution is in so that if you are at the bottom of page and you close window that contains tinemce editor, 
// focus will be moved to first found input field and you will be scrolled to the top for no reason. Not good. IE only.
if ($.browser.msie) {
    setTimeout(function(){
        $('#center-area').focus();
        input.focus();
    },100);
}

> _**Legacy information imported from TinyMCE bug tracker:**_
> 
> _[#T6567](http://archive.tinymce.com/develop/bugtracker_view.php?id=6567) posted by [ajaxel.com](http://archive.tinymce.com/profile.php?name=ajaxel.com)_
> 
> _**Tags:** [msie msie7 msie8 msie9 msie10 ie tinymce focus js crash]_
> _**Status:** Open_
> _**Resolution:** None_
> _**Attached URL:** http://demo.ajaxel.com/_

**Comment by fyrkant** [comment]
Closing this due to old age, if anyone still has this issue in the latest version of TinyMCE please open up a new issue.
